qiskit_metal/renderers/renderer_ansys/ansys_renderer.py

```diff
@@ -1527,7 +1530,6 @@ def set_variables(self, variables: Dict):
             if self.pinfo.design:
                 for k, v in variables.items():
                     self.pinfo.design.set_variable(k, v)
-                    self.pinfo.design.set_variable(k, v)
 
```

Comment:
As we wait for the analysis split merge, how about we add here a "else" to logger.warn people that they need to create a design before they try to set variables, else their rendering will pick up value 0 for all the variables?

Reply:
Done.

Reply:
thank you!